iOS/rainbow/Config/Info.plist

```diff
@@ -14,16 +14,16 @@
 	<string>$(PRODUCT_NAME)</string>
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
-	<key>NSCameraUsageDescription</key>
-	<string>We need camera</string>
-	<key>NSMicrophoneUsageDescription</key>
-	<string>We need microphone</string>
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSCameraUsageDescription</key>
+	<string>We need camera</string>
```

Comment:
Fine for now, but we'll want to have better looking strings for these when we ship :)